spinta/backends/__init__.py

```diff
@@ -529,7 +529,7 @@ def prepare_dtype_for_response(
     action: Action,
     select: dict = None,
 ):
-    assert isinstance(value, (str, int, float, bool, type(None))), (
+    assert isinstance(value, (str, int, float, bool, type(None), type(decimal.Decimal()))), (
```

Comment:
Decimal is not JSON serializable, and should not be added here, instead create a new `prepare_dtype_for_response` command method and convert it to float.

Reply:
isinstance remove